Keele University SAHP.csl

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0">
+  <info>
+    <title>SAHP (Keele University) Enhanced</title>
+    <id>http://www.zotero.org/styles/sahp-keele-university-enhanced</id>
+    <link href="http://www.zotero.org/styles/sahp-keele-university-enhanced" rel="self"/>
+    <author>
+      <name>Custom SAHP Style</name>
+    </author>
+    <category citation-format="author-date"/>
+    <summary>SAHP referencing style including journal articles, books, book chapters, and web pages.</summary>
+    <updated>2025-04-05T00:00:00+00:00</updated>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name form="long" and="text" delimiter=", " initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="short" prefix="In " suffix=": " text-case="capitalize-first"/>
+      <name form="long" and="text" delimiter=", " initialize-with="."/>
+    </names>
+  </macro>
+  <macro name="issued">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="title">
+    <text variable="title" font-style="italic"/>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title"/>
+  </macro>
+  <macro name="access">
+    <group delimiter=". ">
+      <text variable="URL" prefix="Available from: "/>
```

Comment:
please use the `<text term="available at" text-case="capitalize-first"/> and then redefine the term in a custom locale to "available from". See other styles how this is done.